packages/eds-core-react/src/components/Icon/Icon.docs.mdx

```diff
@@ -0,0 +1,69 @@
+
+import { Story,Props } from './../../../.storybook/components'
+import * as icons from '@equinor/eds-icons'
+import { Icon  } from '.'
+
+
+# Icon 
+Component for rendering icons from the `@equinor/eds-icons` package.
+
+<Story id="icons-icon--base" />
+<Props/>
+
+## Installation 
+
+Install the packages in your project directory with your package manager (npm/npm/pnpm).
+```shell
+npm install @equinor/eds-core-react @equinor/eds-icons
+```
+
+## Usage
+
+The `Icon` can then render the icons in two ways: 
+
+### Option 1 
+
+Using the `name` property. This allows you to setup all your icons once and use the `name` property to render it.
+```tsx
+import { Icon } from '@equinor/eds-core-react'
+import { save } from '@equinor/eds-icons' // import "save" icon
+
+Icon.add({ save }) // needs only to be done once
+
+<Icon name="save" />
+```
+### Option 2
+
+Use the `data` property for directly rendering the imported icon
+```tsx
+import { Icon } from '@equinor/eds-core-react'
+import { save } from '@equinor/eds-icons' // import "save" icon
+
+<Icon data={save} />
+```
+
+
+## Accessibility
+If you are using the icons semantically you need to provide the `title` property so that your icons are correctly accessible.
```

Comment:
“Using the icons semantically” should be rephrased, I see we have a similar description in the component itself that should also be updated. The use case is that if you’re using icons alone to convey meaning, you need to provide a title for screen reader users that reflect the action the icon represents. But if you already have icons and text, additional text is superfluous and you should rather use `aria-hidden="true"`.

Reply:
`aria-hidden="true"` is on applied by default and removed when `title` is applied. I'll see if I can reword it somehow.